### PR TITLE
A better message response to forms

### PR DESCRIPTION
**Pull request by thebishaldeb**
Updated response message while saving changes using antd `message` component.

# Description
### Actual Behaviour
Currently when a button is clicked, we do not really know if the changes are taking place until a response message shows up after the execution.

### Expected Behaviour
Now when form will be submitted, a better response will be shown which is taken care of using `message.destroy()` and `message.loading()` components. The message keeps on showing until the completion or failure of action.

# Screenshot of changes :

message

### Checks:

- [x] Tested Localy
- [ ] Checked in IE11 and Firefox
- [ ] Checked in mobile
- [ ] Backend work
- [x] Frontend work
- [ ] Bug Fixes
- [x] Enhancements

### CODE REVIEW

**Comment by thebishaldeb** [comment]
Have a look at it @hsluoyz 
Thanks.

**Comment by hsluoyz** [comment]
Don't let `Merge with upstream.` show up.

Don't modify code style.

Show a demo.

**Comment by thebishaldeb** [comment]
Updated. @hsluoyz Please have a look now.

File: web/src/AdapterPage.js [hunk 1]
```diff
@@ -82,6 +82,7 @@ class AdapterPage extends React.Component {
     //     Setting.showMessage("error", `Sava failed: ${error}`);
     //   });
 
+    Setting.showMessage("loading", `Please wait...`);
```
Comment (hsluoyz) [inline web/src/AdapterPage.js]: Should refactor our a function called `Setting.showWaitingMessage()`.

File: web/src/AdapterPage.js [hunk 2]
```diff
@@ -98,6 +99,7 @@ class AdapterPage extends React.Component {
   }
 
   testAdapterConnection() {
+    Setting.showMessage("loading", `Please wait...`);
```
Comment (hsluoyz) [inline web/src/AdapterPage.js]: Same above

File: web/src/HomePage.js [hunk 3]
```diff
@@ -72,6 +72,7 @@ class HomePage extends React.Component {
   }
 
   updateMetadata() {
+    Setting.showMessage("loading", `Please wait...`);
```
Comment (hsluoyz) [inline web/src/HomePage.js]: Same above

File: web/src/ModelPage.js [hunk 4]
```diff
@@ -111,6 +111,7 @@ class ModelPage extends React.Component {
   }
 
   updateModel() {
+    Setting.showMessage("loading", `Please wait...`);
```
Comment (hsluoyz) [inline web/src/ModelPage.js]: Same above

**Comment by nodece** [comment]
The PR too long, and the new code may fix it. thanks for contribution.

**Pull request closed without merging**